actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -410,6 +410,10 @@ def [](key)
       convert_hashes_to_parameters(key, @parameters[key])
     end
 
+    def to_param
```

Comment:
IIRC we did decide that we intentionally wanted to stop supporting using `params` in URLs without explicit whitelisting (e.g. `slice` and/or `to_h`)